protobuf/api/execution.proto

```diff
@@ -19,37 +23,32 @@ service Execution {
 }
 
 // GetExecutionRequest defines request to retrieve a single execution.
+// **Example**	
+// TODO: add JSON example
```

Comment:
About JSON example I think it's a waste of our time. someone who will use it will know how protobuf is mapped to json.

Reply:
The json example is good in the documentation, it the proto it is not really useful.
I was thinking about this part could be directly injected in the documentation and we could in the file include the `grpcurl` command to test the api and get the data. This way on the proto we have a clear example of how to use it and when we generate the documentation we could actually execute this command to generate with real inputs/outputs and use that as "integration test"

Reply:
```go
package main

import (
  "os"

  "github.com/golang/protobuf/jsonpb"
  "github.com/mesg-foundation/core/protobuf/api"
)

func main() {
  (&jsonpb.Marshaler{EmitDefaults: true, Indent: "\t"}).Marshal(os.Stdout, &api.GetExecutionRequest{})
}
```

Just found the Marshaler, maybe it could be integrated into docs creation pipeline if we want to have it generated.

Reply:
ok i will remove json example